### ISSUE TITLE
[RA1] Remove Introduction section header

### ISSUE THREAD

**Issue by pgoyal01** [cntt-n/CNTT#3033]
Remove Introduction section header from all but Chapter 1

**Comment by collivier** [comment]
@pgoyal01 we can use one introduction per chapter.

**Comment by pgoyal01** [comment]
@collivier Yes -- but given that there is nothing preceding does it make sense to have th Introduction section?

**Comment by collivier** [comment]
agree
